### ISSUE TITLE
Logistic regression model has same f1 score over all sample

### ISSUE THREAD

**Issue by alokpadhi** [alokpadhi/Student-Intervention-System#1]
When I trained my logistic regression model, for all sample sized in the test set the f1 score is same. Which seems unnatural. Beforehand I have applied StandardScaler() to fit in the same scale. Notable standardize fit on the training set and using that I have transformed the test set.
